DAL/user.DAL.js

```diff
@@ -0,0 +1,42 @@
+// get all employees -- need to move ot DAL
+getAllUsers = () => {
+    return {1 : "Gilad", 2 : "Adva", 3: "Ofir", 4: "Dave"};
+};
+
+const getUserById = (id) => {
+    if(id == null)
+        return 'No valid id';
+    let users = getAllUsers();
+    if (id in users) {
+        return users[id];
+    }
+    return ["This id doesn't exist"];
```

Comment:
`["This id doesn't exist"]`
returning an error string inside an Array is not a common approach. please handle errors as seen in class, if you need some further explanation, please LMK